### PR TITLE
feat(web): add image push and pull to notifications

### PR DESCRIPTION
**Pull request by m8vago**
Add new notification types:
- Image push
- Image pull

Add an option for creating jwt tokens for v2 registries, so we can receive notifications from them.

### CODE REVIEW

File: web/crux/src/app/registry/guards/registry.auth.validation.guard.ts [hunk 1]
```diff
@@ -134,7 +134,7 @@ export default class RegistryAccessValidationGuard implements CanActivate {
     const auth = await this.getCredentialsForRegistry(registryId, req)
 
     return this.httpService
-      .get(`https://${req.url}/v2/`, {
+      .get(`http://${req.url}/v2/`, {
```
Comment (polaroi8d) [inline web/crux/src/app/registry/guards/registry.auth.validation.guard.ts:137]: Is this a real change?

File: web/crux/src/app/registry/registry-clients/v2-api-client.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ class RegistryV2ApiClient implements RegistryApiClient {
 
   private async fetch(endpoint: string, init?: RequestInit): Promise<Response> {
     const initializer = init ?? {}
-    const fullUrl = `https://${this.url}/v2${endpoint}`
+    const fullUrl = `http://${this.url}/v2${endpoint}`
```
Comment (polaroi8d) [inline web/crux/src/app/registry/registry-clients/v2-api-client.ts:79]: Same question here.

**Review by polaroi8d — CHANGES_REQUESTED** [review]

File: web/crux/src/app/registry/registry-clients/v2-api-client.ts (hunk 2 above)
Comment (m8vago) [reply web/crux/src/app/registry/registry-clients/v2-api-client.ts:79]: Nope. Nice catch!

**Review by polaroi8d — APPROVED** [review]
Okey for me, pleas @robot9706 if you have time check it please.

File: web/crux/prisma/migrations/20231019133412_deploy_log/migration.sql [hunk 3]
```diff
@@ -1,4 +1,15 @@
---Update
-UPDATE "DeploymentEvent"
```
Comment (robot9706) [inline web/crux/prisma/migrations/20231019133412_deploy_log/migration.sql:2]: Why this older migrations is modified?

**Review by robot9706 — COMMENTED** [review]
Sesms fine to me, but couldn't test it. One comment on the migration though.

**Review by robot9706 — APPROVED** [review]

**Pull request merged**